pyxem/utils/calibration_utils.py

```diff
@@ -0,0 +1,122 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-2021 The pyXem developers
+#
+# This file is part of pyXem.
+#
+# pyXem is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# pyXem is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with pyXem.  If not, see <http://www.gnu.org/licenses/>.
+
+
+import numpy as np
+from pyxem.utils.indexation_utils import index_dataset_with_template_rotation
+from diffsims.generators.library_generator import DiffractionLibraryGenerator
+
+def find_diffraction_calibration(
+    patterns,calibration_guess,library_phases,lib_gen,size,max_excitation_error = 0.01, **kwargs
```

Comment:
New code in pyxem should adher to the black code style. You can enforce this on this file by `pip install black` and `black calibration_utils.py`.